xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}	$csv{$query,1}	$hitlist{$id$.}	$topdoc}
```

Comment:
> There's also something odd (and not necessarily correct) about using $csv{} when you're generating a TSV. 

Using`$csv{}` here just to quote query strings.

> TSV is flimsily specified, perhaps even more so than CSV :-(

I think we have to use TSV format here because there could be `,` in query strings which will confuse postprocessing script leading to incorrect log records. Is there a better way to specify TSV here?